classes/GAdwordsModuleManagement.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * 2007-2019 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/afl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ *  @author    PrestaShop SA <contact@prestashop.com>
+ *  @copyright 2007-2019 PrestaShop SA
+ *  @version	Release: $Revision: 17142 $
+ *  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+ *  International Registered Trademark & Property of PrestaShop SA
+ */
+
+class GAdwordsModuleManagement
+{
+    /**
+	 * Manage the module and return the module configuration link if it is installed
+	 *
+	 * @return bool
+	 */
+	public function moduleManagement($moduleName, $moduleId)
+	{
+		// Is the module is installed ?
+		if (Module::isInstalled($moduleName)) {
+			return true;
+		}
+
+		$this->isModuleOnDisk($moduleName, $moduleId);
+
+		$modulePrestashopAds = Module::getInstanceByName($moduleName);
+		
+		if ($modulePrestashopAds->install()) {

```

Comment:
```suggestion
		return $modulePrestashopAds->install();
```

Then you can remove the next lines

Reply:
Thanks for the help.